src/templates/modals/add_scheduled_run.html

```diff
@@ -0,0 +1,286 @@
+<button type="button" class="btn btn-outline-primary" data-toggle="modal" data-target="#sched-run-modal" onclick="addScheduledRun()">
```

Comment:
Does this need to be styled to have a fixed width to match the other buttons?

Reply:
Yea that's needed. Actually instead of using hardcoded style at each button, can we write a class for it? It's good practice to use class instead of hard coded styles.

Reply:
That's probably a good idea, where would we put styles though?

Reply:
Anywhere in the html is fine with me. Something like
```
<style>
  .class-name {
    width: xxx px;
  }
</style>
```

Reply:
Ah, alright! Wasn't sure if there was a dedicated stylesheet. Thank you!